api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -829,7 +823,7 @@ public AuthenticationResponse begin() {
      */
     public AuthenticationResponse beginPasswordRecovery(String token) {
```

Comment:
We should probably expose the setting of the DeviceContext here too.
From the audit perspective, the user agent/IP of the client is important

Reply:
Though I don't know if this method is used to start a new transaction or not. (if it isn't ignore this comment)

Reply:
Yes, it starts a new transaction with a `AuthenticationTransaction.create(..)`.